crypto/conf/keysets.pl

```diff
@@ -107,22 +103,21 @@
 #define IS_ALNUM_PUNCT(c,a) (KEYTYPES(c)[CVT(a)] & CONF_ALNUM_PUNCT)
 #define IS_QUOTE(c,a)       (KEYTYPES(c)[CVT(a)] & CONF_QUOTE)
 #define IS_DQUOTE(c,a)      (KEYTYPES(c)[CVT(a)] & CONF_DQUOTE)
-#define IS_HIGHBIT(c,a)     (KEYTYPES(c)[CVT(a)] & CONF_HIGHBIT)
 
 EOF
 
 my $i;
 
-print "static const unsigned short CONF_type_default[256] = {";
-for ($i = 0; $i < 256; $i++) {
+print "static const unsigned short CONF_type_default[128] = {";
+for ($i = 0; $i < 128; $i++) {
     print "\n   " if ($i % 8) == 0;
-    printf " 0x%04X,", $V_def[$i];
+    printf " 0x%03X,", $V_def[$i];
```

Comment:
This is purely a cosmetic change and I don't see the need for it.  It just makes the diff needlessly bigger.

Reply:
Ok, on the basis that this is trivial and purely cosmetic I have left this.